keps/993-two-phase-admission/README.md

```diff
@@ -232,8 +261,15 @@ are still valid (check quota/preemptions/reclamation) and admit the workload (do
 newly identified preemptions, if needed), setting the `Admitted`
 condition to True.
 
-If any check is switched to "False", with reason "Reject" or "Retry", the workload is either
-completely rejected or switched back to suspend state (with retry delay).
+If a ProvisioningRequest check is switched to `Retry` state, the Workload will be requeued
+or deactivated, with respect to the `ProvisioningRequestRetryStrategy.defaultRequeuingStrategy` defined in
+Kueue's global Configuration. If the Workload is requeued, controller should update Workload's `.status.requeueState`
```

Comment:
>  If the Workload is requeued, controller ...

Which Controller should do this? the workload controller or the admission check controller.